ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/test/java/de/adorsys/ledgers/baam/db/repository/ManagerAccessRepositoryIT.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2018-2023 adorsys GmbH and Co. KG
+ * All rights are reserved.
+ */
+
+package de.adorsys.ledgers.baam.db.repository;
+
+import de.adorsys.ledgers.baam.db.domain.AccessScope;
+import de.adorsys.ledgers.baam.db.domain.AccessStatus;
+import de.adorsys.ledgers.baam.db.domain.ManagerAccess;
+import de.adorsys.ledgers.baam.db.domain.TypeOfManagedAccess;
+import de.adorsys.ledgers.baam.db.test.BaamRepositoryApplication;
+import jakarta.transaction.Transactional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.Rollback;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(classes = BaamRepositoryApplication.class)
+@Transactional // Ensures that each test is executed within a transaction and rolled back afterwards
+@Rollback // Reverts the changes made to the database after each test
+public class ManagerAccessRepositoryIT {
+
+
+    @Autowired
+    private ManagerAccessRepository managerAccessRepository;
+
+    // Variable to store a ManagerAccess object
+    private ManagerAccess managerAccess;
+
+    @BeforeEach
+    public void setup() {
+        // Initialize a ManagerAccess object for testing
+        managerAccess = new ManagerAccess();
```

Comment:
These is correct but not a good practice for our case..

it should be  `ManagerAcess  managerAccess = new ManagerAccess(); ` and not ` managerAccess = new ManagerAccess();`